examples/bfetch_explorer/public/hooks/use_deps.ts

```diff
@@ -9,4 +9,4 @@
 import { useKibana } from '@kbn/kibana-react-plugin/public';
 import { BfetchDeps } from '../mount';
 
-export const useDeps = () => useKibana().services as BfetchDeps;
+export const useDeps = () => useKibana().services as unknown as BfetchDeps;
```

Comment:
NIT: I have no idea why the PR make TS complain about these two example plugin files, but I had to coerce to `unknown` here.